tests/5.0/teams/test_teams.F90

```diff
@@ -0,0 +1,75 @@
+!===--- test_teams.F90 -----------------------------------------------------===//
+!
+! OpenMP API Version 5.0 Nov 2018
+!
+! This test uses the teams directive on host and verifies the 
+! requested number of teams with requested number of threads were
+! created.
+!
+!//===----------------------------------------------------------------------===//
+
+#include "ompvv.F90"
+
+#define N 4
+#define K 2
+
+PROGRAM test_teams
+ 
+  USE iso_fortran_env
+  USE ompvv_lib
+  USE omp_lib
+  implicit none
+
+  OMPVV_TEST_OFFLOADING
+
+  OMPVV_TEST_VERBOSE(teams() .ne. 0)
+
+  OMPVV_REPORT_AND_RETURN()
+
+CONTAINS
+    INTEGER FUNCTION teams()
+    CHARACTER(len=300):: infoMsg
+    INTEGER,DIMENSION(N):: num_teams, num_threads, a, b
+    INTEGER,DIMENSION(K):: errors
+    INTEGER:: x
+    errors(1) = 0
+    errors(2) = 0
+
+    DO x = 1, N
+      num_teams(x) = -99
+      num_threads(x) = -99
+    END DO
+
+    !$omp teams num_teams(N) thread_limit(N)
```

Comment:
Same thing, can use OMPVV_NUM_TEAMS_DEVICE and OMPVV_NUM_THREADS_HOST here. Altho because it is a standalone teams we may need a OMPVV_NUM_TEAMS_HOST macro? I can add this if we think it is needed.